----------------------------------------------------------------------
Revision: d31e3528a05a44cb37564f06484906872f10eb5d
Parent:   2ed09e2720ad78b81a36d3b3d90602e8459b58b8
Author:   yan12125@gmail.com
Date:     西元2019年06月13日 21時19分45秒
Branch:   net.venge.monotone

Changelog: 

Fix building on Mac OS X 10.12

Changes against parent 2ed09e2720ad78b81a36d3b3d90602e8459b58b8

  patched  src/gzip.cc

============================================================
--- src/gzip.cc	7872a4ddd5d805731cfaba0a7ff743224763f3fc
+++ src/gzip.cc	f6e5c2af3852c44eae2c7f3315ad124f86864455
@@ -18,6 +18,7 @@
 
 #include "botan_glue.hh"
 #include <cstring>
+#include <cstdlib>
 #include <map>
 #include <zlib.h>
 
